packages/preset-mini/src/autocomplete.ts

```diff
@@ -5,6 +5,6 @@ export const autocomplete: AutoCompleteTemplate[] = [
   '(border|b)-(rounded-|rd-)#num',
   '(border|b)-$colors',
   '(m|p)(x|y|t|b|l|r|s|e|)-#num',
-  'text-$colors',
+  'text-$colors:deep',
```

Comment:
We probably don't need the `:deep`, we could always going deep by check if the child is an object or not

Reply:
Sure, could be done by default.